lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+* *Validator Node* is a replica that runs the Libra Protocol to maintain the blockchain state. A Validator Node processes transactions and interacts with other Validator Nodes to reach consensus on the ordering of transactions and the resulting state of the database after transactions' executions.
+* *Validator Operator* is an owner of an on-chain Libra account, it may operate none, one or several Validator Nodes: a Validator Owner may designate a Validator Operator to run a Validator Node on Validator Owner's behalf. A Validator Operator may run multiple Validator Nodes. The Validator Operator for a Validator Owner has permission to rotate the Validator Config for the Validator Owner locally, as well as copy it to the Validator Set.
+* *Reconfiguration* defines a new epoch by triggerring a new epoch event (`LibraConfig::NewEpochEvent`), it gets emitted if any part of the global on-chain configuration, such as the Validator Set, changes. If a transaction causes an epoch change, that transaction is the last transaction committed in the current epoch — any subsequent transactions in that block or future blocks from that epoch will be ignored. Once the transaction has been committed, the new configuration starts being valid immediately. E.g. the new Validator Set will start running the consensus protocol in the new epoch.
+
+<img alt="Validator Set" src={useBaseUrl('img/validator_set.svg')} />
+
+# Specifications
+
+## Workflows
+### Validator Owner and Validator Operator accounts creation
+Only the Association (through the Association Operation Services) may create Validator Owner and Validator Operator accounts.
+Validator Owners and Validator Operators should contact the Association off-chain and communicate their authentication keys as well as the names of their organizations written in snake case (lowercase, where spaces are replaced with underscores). The authentication key is a 32 elements byte array, derived from the public verification key. The protocol currently supports Ed25519 and MultiEd25519 signature schemes.
+
+For Validator Owner the Libra Account has a *Validator Role* (see [lip-2: Libra Roles and Permissions](https://github.com/libra/lip/blob/master/lips/lip-2.md)) and stores a `ValidatorConfig` resource managed by `ValidatorConfig` module:
+
+```
+resource struct ValidatorConfig {
+  config: Option<Config>,
+  operator_account: Option<address>,
+  human_name: vector<u8>,
+}
+```
+It contains a `human_name` of the account which is set at the account creation time and cannot be changed. The `config` and `operator_account` fields are initially set to `none`. The `config` is a place holder for parameters of the Validator Node.
+```
+struct Config {
+  consensus_pubkey: vector<u8>,
+  validator_network_addresses: vector<u8>,
+  fullnode_network_addresses: vector<u8>,
+}
+```
+
```

Comment:
we should evaluate if we want these fields to be lcs encoded... as of now ... they are technically in a grey area. cc: @zekun000